monolith/client/__init__.py

```diff
@@ -133,22 +140,19 @@ def __call__(self, field, start, end, interval=DAY, strict_range=False,
 
         if terms:
 
-            range_ = query['facets']['histo1']['facet_filter']['range']
+            range_ = query['aggs']['ranged']['filter']['range']
 
-            query['facets']['histo1']['facet_filter'] = {
+            query['aggs']['ranged']['filter'] = {
                 'and': ([{'term': {k: v}} for k, v in terms.items()] +
                         [{'range': range_}])}
 
         res = self.raw(query)
         counts = {}
 
-        for entry in res['facets']['histo1']['entries']:
-            time_ = entry['time'] / 1000.0
+        for entry in res['aggregations']['ranged']['histo']['buckets']:
```

Comment:
No, IIRC `aggs` is an alias for `aggregations` that you can use for the query, but in the response it's always `aggregations`.